edx_notifications/renderers/basic.py

```diff
@@ -74,11 +77,17 @@ def render(self, msg, render_format, lang):
                 self.underscore_template = us.template(template_string)
 
         _payload = copy.deepcopy(msg.payload)
+
+        if datetime.datetime.date(msg.created) == datetime.datetime.now(pytz.UTC).date():
+            created_str = 'Today at ' + msg.created.strftime("%H:%M%p")
+        else:
+            created_str = msg.created.strftime("%B %d, %Y") + ' at ' + msg.created.strftime("%H:%M%p")
```

Comment:
I think we shouldn't translate this to today mainly because all datetime's are in GMT and this could look a little funny to users. I wrote up a story whether we should translate digest timestamps to localtime (based on a browser telling us what timezone the user last logged in at). But this needs to be priorities.

So, in the meantime, remove the 'if' statement and not use the 'Today' notation.